packages/mobile/src/account/actions.ts

```diff
@@ -149,6 +154,7 @@ export interface SetRetryVerificationWithFornoAction {
 export interface ClearStoredAccountAction {
   type: Actions.CLEAR_STORED_ACCOUNT
   account: string
+  softDelete: boolean
```

Comment:
Making the naming more explicit?
```suggestion
  onlyReduxState: boolean
```